architecture/adr-list/adr-005-genesis-parameters.md

```diff
@@ -0,0 +1,167 @@
+
+### Consensus parameters
+Tendermint requires [genesis parameters](https://docs.tendermint.com/master/tendermint-core/using-tendermint.html#genesis) to be defined for basic consensus conditions on any Cosmos network.
+
+#### Proposed values
+
+* **`block`**
+  * `max_bytes` = `200000` \(~200KB\)
+    * Cosmos hub: `200000` \(~200KB\)
+  * `max_gas` = `2000000` (~20 txs)
+    * Cosmos hub: `2000000` (~20 txs)
+  * `time_iota_ms` = `1000` (1s)
+    * Cosmos hub: `1000` (1s)
+    * **Deprecated, unused**
+* **`evidence`**
+  * `max_age_num_blocks` = `1576800`
+    * Maximum age of evidence, in blocks. The basic formula for calculating this is: `MaxAgeDuration / {average block time}`.
+  * `max_age_duration` = `7884000`
+    * Maximum age of evidence, in time. It should correspond with an app's "unbonding period".
+  * `max_bytes` = `1048576`
+    * This sets the maximum size of total evidence in bytes that can be committed in a single block and should fall comfortably under `max_bytes` for a block.
+* **`validator`**
+  * `pub_key_types` = `[ "ed25519" ]`
+
+### Application parameters
+
+Cosmos application is divided [into a list of modules](https://docs.cosmos.network/v0.44/modules/). Each module has parameters that help to adjust the module's behaviour. Here are proposed values for default modules:
+
+* **`auth`**
+  * `max_memo_characters` = `512`
+    * Maximum number of characters in the memo field
+  * `tx_sig_limit` = `7`
+    * Max number of signatures
+  * `tx_size_cost_per_byte` = `10`
+    * Gas cost of transaction byte
+  * `sig_verify_cost_ed25519` = `590`
+    * Cost of `ed25519` signature verification
+  * `sig_verify_cost_secp256k1` = `1000`
+    * Cost of `secp256k1` signature verification
+* **`bank`**
+  * s`end_enabled` = `[]`
+    * Enables send for specific denominations
+  * `default_send_enabled` = `true`
+    * The default send enabled value allows send transfers for all coin denominations
+* **`crisis`**
+  * `constant_fee` = `{ "denom": "ncheq", "amount": "10.000.000.000.000" }`
+    * The fee is used to verify the [invariant(s)](https://docs.cosmos.network/v0.44/building-modules/invariants.html) in the `crisis` module.
+* **`distribution`**
+  * `community_tax` = `0.02`
+    * The percent of rewards that goes to the community fund pool
+  * `base_proposer_reward` = `0.01`
+    * Base reward that proposer gets
+  * `bonus_proposer_reward` = `0.04`
+    * Bonus reward that proposer gets. This depends on the number of pre-commits included to the block
+  * `withdraw_addr_enabled` = `true`
+    * Whether withdrawal address can be changed or not. By default, it's the delegator's address.
+* **`evidence`**
+  * No parameters
+* **`genutil`**
+  * Used to manage initial transactions such as genesis validators creation
+* **`gov`**
+  * `deposit_params`
+    * min_deposit = `[{ "denom": "ncheq", "amount": "8.000.000.000.000" }]`
+      * The minimum deposit for a proposal to enter the voting period.
```

Comment:
Same comment as above on decimal vs comma separators. Also is the number of zeros correct? If min deposit is 8000 cheq, this feels incorrect.